### PR TITLE
Translate glossary for ConfigMap into Bahasa

### PR DESCRIPTION
**Pull request by habibrosyad**
/cc @ariscahyadi @girikuncoro 

Note: I deliberately make 2 commits for this PR to test the new config [here](https://github.com/kubernetes/website/issues/24043). For whoever reviewing this, after finished with the review (and if it is acceptable), please give lgtm, but do not approve until all commits have been squashed into one. Thanks 😄

### CODE REVIEW

**Comment by habibrosyad** [comment]
Preview: https://deploy-preview-24096--kubernetes-io-master-staging.netlify.app/id/docs/reference/glossary/?core-object=true#term-configmap

File: content/id/docs/reference/glossary/configmap.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: ConfigMap
+id: configmap
+date: 2018-04-12
+full_link: /docs/concepts/configuration/configmap/
+short_description: >
+  Sebuah objek API yang digunakan untuk menyimpan data nonkonfidensial sebagai pasangan kunci-nilai (_key-value_). Pod dapat menggunakan ConfigMap sebagai variabel lingkungan, argumen baris perintah (_command-line_), atau berkas konfigurasi dalam sebuah _volume_.
```
Comment (girikuncoro) [inline content/id/docs/reference/glossary/configmap.md:7]: nice, another new word I learned https://kbbi.kemdikbud.go.id/entri/konfidensial

**Review by ariscahyadi — COMMENTED** [review]
Nice work @habibrosyad !
/lgtm

**Comment by habibrosyad** [comment]
Squashed! and the lgtm is still here, thanks @girikuncoro @ariscahyadi. If there are no issue with this PR, I think it is ready to be approved anytime.

/assign @girikuncoro

**Pull request merged**